image/icc_4.ksy

```diff
@@ -96,7 +96,7 @@ types:
         0x6172676C: argyll_cms_cmm #argl
         0x44676f53: logosync_cmm #LgoS
         0x48444d20: heidelberg_cmm #HDM
-        0x6C636d73: little_cms_cmm #lcms
+        0x6C636D73: little_cms_cmm #lcms
```

Comment:
While you do this change, please also normalize the other enum values:

https://github.com/kaitai-io/kaitai_struct_formats/blob/e14dc4a2ed47609e2f33798670819d0d3591fa0c/image/icc_4.ksy#L95-L109

```diff
-        0x48434d4D: harlequin_rip_cmm #HCMM
+        0x48434D4D: harlequin_rip_cmm #HCMM
         0x6172676C: argyll_cms_cmm #argl
-        0x44676f53: logosync_cmm #LgoS
+        0x44676F53: logosync_cmm #LgoS
-        0x48444d20: heidelberg_cmm #HDM
+        0x48444D20: heidelberg_cmm #HDM
         0x6C636D73: little_cms_cmm #lcms
-        0x4b434d53: kodak_cmm #kcms
+        0x4B434D53: kodak_cmm #kcms
-        0x4d434d44: konica_minolta_cmm #MCML
+        0x4D434D44: konica_minolta_cmm #MCML
         0x57435320: windows_color_system_cmm #WCS
         0x5349474E: mutoh_cmm #SIGN
-        0x52474d53: device_link_cmm #RGMS
+        0x52474D53: device_link_cmm #RGMS
         0x53494343: sample_icc_cmm #SICC
-        0x54434d4d: toshiba_cmm #TCMM
+        0x54434D4D: toshiba_cmm #TCMM
         0x33324254: the_imaging_factory_cmm #32BT
         0x57544720: ware_to_go_cmm #WTG
-        0x7a633030: zoran_cmm #zc00
+        0x7A633030: zoran_cmm #zc00
```

https://github.com/kaitai-io/kaitai_struct_formats/blob/e14dc4a2ed47609e2f33798670819d0d3591fa0c/image/icc_4.ksy#L284

https://github.com/kaitai-io/kaitai_struct_formats/blob/e14dc4a2ed47609e2f33798670819d0d3591fa0c/image/icc_4.ksy#L1658

Reply:
Thanks for mentioning that. I have missed this.